Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityControllerVisualizationProfileInspector.cs

```diff
@@ -56,8 +58,10 @@ protected override void OnEnable()
             renderMotionControllers = serializedObject.FindProperty("renderMotionControllers");
             defaultControllerVisualizationType = serializedObject.FindProperty("defaultControllerVisualizationType");
             useDefaultModels = serializedObject.FindProperty("useDefaultModels");
```

Comment:
Is this still the correct serialized property name? This changed with gltf changes in the public repo.

Reply:
if not, there may have been merge was incorrectly resolved. we should investigate and update as a separate PR if necessary